src/test/java/org/opentripplanner/apis/gtfs/GraphQLIntegrationTest.java

```diff
@@ -196,8 +197,20 @@ static void setup() {
       .toList();
 
     var busRoute = routes.stream().filter(r -> r.getMode().equals(BUS)).findFirst().get();
+    TransitEditorService transitService = new DefaultTransitService(transitModel) {
+      private final TransitAlertService alertService = new TransitAlertServiceImpl(transitModel);
```

Comment:
Not sure what we should do here - this is a bit ugly. We do not want to manually wire tests, but in this case we want to mock part of the API, so it is difficult to avoid it. Maybe we should revisit this after the "dependency injection api scope problem" is fixed(PR #5442).